src/main/target/WORMFC/target.h

```diff
@@ -0,0 +1,133 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define TARGET_BOARD_IDENTIFIER "RRF4"
+#define USBD_PRODUCT_STRING     "Worm FC"
+
+//LEDs
+#define LED0_PIN                PA15
+#define LED1_PIN                PC14
+
+//BEEPER
+#define BEEPER                  PB14
+#define BEEPER_INVERTED
+
+// MPU6500 interrupt
+#define USE_EXTI
+#define MPU_INT_EXTI            PC4
+#define USE_MPU_DATA_READY_SIGNAL
+//#define DEBUG_MPU_DATA_READY_INTERRUPT
+
+#define MPU6500_CS_PIN          PA4
+#define MPU6500_SPI_INSTANCE    SPI1
+
+// ACC section -- start
+#define ACC
+#define USE_ACC
+#define USE_ACC_MPU6500
+#define USE_ACC_SPI_MPU6500
+#define ACC_MPU6500_ALIGN       CW180_DEG_FLIP
+// ACC section -- end
+
+// GYRO section -- start
+#define GYRO
```

Comment:
Defines for `ACC`, `GYRO`, `BARO` should not be in here - adding bogus defines to target definitions just complicates the work of the maintainers.